extension/lib/doc.js

```diff
@@ -7,6 +7,9 @@ if (window[namespace] === true) {
     window[namespace] = true;
 }
 
+// If the number of sentences in the doc exceeds this, mark the document as unreadable.
+const MAX_NUM_SENTENCE = 1000;
+
```

Comment:
It'd be interesting to see how long an article would be for this to be the case; for instance, what about a long Wikipedia article? An online book? Also related to speed function, i.e. the function time_to_load(#_of_sentences)